### ISSUE TITLE
Language Support

### ISSUE THREAD

**Issue by sfscgjohnson** [salesforce-experiencecloud/KnowledgeArticleBodyContent#1]
Is there any language support for this component?

**Comment by gtabboud** [comment]
Yes, I just tested this in my org, and it does return the translation based on the user's language or the guest user's selected language from the picker.

**Comment by sfscgjohnson** [comment]
Ok so I did what you said and added the picker.  In my dev box this worked.
As soon as I move it to production, it is only showing German on my LWR site and German isn't even one of the language selected.  Weirder still Google says it is in the Language I selected but when attempting to translate to English.  It still shows in German.

**Comment by sfscgjohnson** [comment]
Ok so we believe that we discovered that the language selector is irrelevant the code appears to only be grabbing the last published article switching language doesn't change the code nor does it enable it to grab any other article.  so if the last one you did was in German even though the site doesn't support German nor is it selected it still displays German.  I think this is a show stopper bug. :-(

No where in getArticle does it check the language so it is just getting the first knowledge article that was input.
It wouldn't know to get the version in the language you want.

Unless I am missing something.

**Comment by gtabboud** [comment]
I don't believe that's what's happening. I setup a quick test at this url: https://devfull-learncommunitycloud.cs18.force.com/s/article/Declarative-Deal-Registration-Tips-Tricks?language=en_US

If you change the language to French, you'll see the French translation. If you switch to english, it's the english translation. Seems to be working as expected here.

**Comment by sfscgjohnson** [comment]
This is an all LWR site I cannot tell if yours is one also.
My path is URL/article no /s/
When I got to it I'd go to https:/URL/article/?urlName=1417 and it loads the article in the last language it was published in.

If I try to do the pathing that you did (without the /s/)  it doesn't show anything.  If I switch the language it always shows that last language.  Even though the site itself doesn't have the language supported.   Eg showing German when I haven't selected German as an option for my LWR site.

**Comment by sfscgjohnson** [comment]
Also, I'd like to point out that when I change the language it changes the URL to https://URL/lang/article?urlName=1417
If I try to put the URL as you did https://URL/article?urlName=1417&language=ja  or any language it still shows the same German or Polish article (Polish for the one where I last published Polish)
